packages/components/src/list/stories/index.js

```diff
@@ -35,7 +40,32 @@ function logItemClick( event ) {
 export default {
 	title: 'WooCommerce Admin/components/List',
 	component: List,
-	decorators: [ ( storyFn, context ) => withConsole()( storyFn )( context ) ],
+	decorators: [
+		( storyFn, context ) => withConsole()( storyFn )( context ),
+		withLinks,
+	],
+	parameters: {
+		docs: {
+			page: () => (
+				<>
+					<Title />
+					<Subtitle />
+					<Description
+						markdown={ `[depreciated] and will be replaced by
```

Comment:
Should be *deprecated

Reply:
Thanks should be fixed in https://github.com/woocommerce/woocommerce-admin/pull/6986/commits/0d715559ddd3eda1e3079d778af2a6112cc6a5e8